nomics/src/endpoint/globalmarketcap.ts

```diff
@@ -0,0 +1,31 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
+
+export const Name = 'globalmarketcap'
+
+const customError = (data: any) => data.Response === 'Error'
+
+export const execute = async (config: Config, request: AdapterRequest) => {
+  const validator = new Validator(request, {})
+  if (validator.error) throw validator.error
+  const jobRunID = validator.validated.id
+
+  const params = {
+    key: config.apiKey,
+  }
+
+  const reqConfig = {
+    ...config.api,
+    baseURL: `https://api.nomics.com/v1/global-ticker`,
```

Comment:
We could include the Base url in config, instead of the ticker, and then on each endpoint, include its corresponding url.
I guess `baseUrl` should be `'https://api.nomics.com/v1/`, and then 
- globalMarketCap -> `url: "global-ticker"`
- price -> `url: "/currencies/ticker"`